apps/console/src/index.tsx

```diff
@@ -96,6 +96,24 @@ ReactDOM.render(
     document.getElementById("root")
 );
 
+
+ReactDOM.render(
```

Comment:
Since we have the providers and router declared in the root element, I feel like this might duplicate them in the DOM. WDYT?

Reply:
We can actually remove the router. The rest are needed. And, if I am not wrong, this is the only option we have since Semantic applies a blur filter to the root element as a whole.

Reply:
Semantic's Message component is now used to display notifications within the modal instead of using the alert component.